boards/nrf52840dongle/Makefile.features

```diff
@@ -0,0 +1,10 @@
+CPU_MODEL = nrf52840xxaa
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_usbdev
```

Comment:
```suggestion
FEATURES_PROVIDED += periph_usbdev
FEATURES_PROVIDED += periph_rtt
FEATURES_PROVIDED += periph_timer
```

Reply:
This should be already set via https://github.com/RIOT-OS/RIOT/blob/d12de7935589581fef1f748a49506a5930bcae80/boards/common/nrf52/Makefile.features#L4-L5